features/google/src/model/importers/activities-importer.js

```diff
@@ -7,12 +9,18 @@ class ActivityParser {
         document.body.appendChild(this._iframe);
     }
 
-    _scrapeTimestamps(contentDocument) {
+    _scrapeTimestamps(contentDocument, productName) {
         const contentCells = contentDocument.querySelectorAll(
             ".mdl-grid>.mdl-cell>.mdl-grid>.content-cell:nth-child(2)"
         );
         return [...contentCells].map(
-            ({ childNodes }) => childNodes[childNodes.length - 1].textContent
+            ({ childNodes }) =>
+                new UserActivity({
+                    timestamp: new Date(
+                        childNodes[childNodes.length - 1].textContent
+                    ),
+                    productName: productName,
```

Comment:
`productName` is just used here. Would this possibly be better where the result is received? Also, UserActivity is imported with this sole purpose.

Reply:
`productName` is initialised there and will be used for example in the ministory. Here `UserActivity` could also be independent on how the data is extracted. Now we extract data from html. Later on we could extract data from the json file and in both cases populate the same type of entity.